### PR TITLE
Fix lightweight themes data in search API

### PR DESCRIPTION
**Pull request by diox**
Fix mozilla/addons-frontend#1964

### CODE REVIEW

File: src/olympia/addons/serializers.py [hunk 1]
```diff
@@ -421,7 +421,9 @@ def fake_object(self, data):
                     display_username=persona_data['author'],
                     header=persona_data['header'],
                     footer=persona_data['footer'],
-                    persona_id=1 if persona_data['is_new'] else None,
+                    # "New" Persona do not have a persona_id, it's an old relic
```
Comment (muffinresearch) [inline src/olympia/addons/serializers.py]: Miniscule nit:

s/it's an old relic from old ones/it's a relic from old ones/

**Comment by muffinresearch** [comment]
r+

File: src/olympia/addons/serializers.py [hunk 2]
```diff
@@ -421,7 +421,9 @@ def fake_object(self, data):
                     display_username=persona_data['author'],
                     header=persona_data['header'],
                     footer=persona_data['footer'],
-                    persona_id=1 if persona_data['is_new'] else None,
+                    # "New" Persona do not have a persona_id, it's a relic from
+                    # old ones.
+                    persona_id=0 if persona_data['is_new'] else 42,
```
Comment (eviljeff) [inline src/olympia/addons/serializers.py]: 42? Eh?

Comment (diox) [inline src/olympia/addons/serializers.py]: Any non-zero value will work, that id is useless, it's only used to determine what `Persona.is_new()` should return.

Comment (diox) [inline src/olympia/addons/serializers.py]: (Ideally there should just be a boolean field called `is_new` in `Persona` to make things less confusing...)

Comment (eviljeff) [inline src/olympia/addons/serializers.py]: and `1` is too boring? 😃 
Ideally all those image files should be [converted](#2695) so we don't have to have an `is_new` property at all.

Comment (diox) [inline src/olympia/addons/serializers.py]: `1` was not remarkable enough, `42` denotes it's a hack :) Yeah, converting the files would be a much better solution I agree.

**Pull request merged**